internal/daemonset/daemonset.go

```diff
@@ -14,7 +15,8 @@ import (
 )
 
 const (
-	imageWithVersion                   = "quay.io/testnetworkfunction/debug-partner:latest"
+	imageWithVersion                   = os.Getenv("TNF_PARTNER_REPO") != "" ? 
```

Comment:
Good catch! A couple of comments: there's no ternary operator in golang "?", so you'll need to create or use an existing function to set this package variable. Also, The image name should also be optionally set by another env var,[ like in the old yaml](https://github.com/test-network-function/cnf-certification-test-partner/blob/d250d1acf1091e82efd6e36e2100594eabc1975c/test-partner/debugpartner.yaml#L25) (SUPPORT_IMAGE).

Reply:
Sure, I'll check it now, I just started the change yesterday to review it with more depth today. I'll keep you posted

Reply:
Done, I've included this in my last commit. I'll start testing that on DCI.